server/plugins/engine.ts

```diff
@@ -66,21 +66,22 @@ function startEngine({ botApi }: StartEngineArgs) {
     bots = newBots;
   });
 
+  const timerInterval = setInterval(() => {
```

Comment:
nit: давай использовать немного более конкретное название для переменной

```suggestion
  const gameTimeoutInterval = setInterval(() => {
```